### ISSUE TITLE
Staging Review finding: No focus management between pages

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98481]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IIR
**Product:** Veteran New Account Flow within Authenticated Experience
**Feature:** Welcome Pathway V1

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When a screen reader user presses Continue on [/review-information/contact-information](https://staging.va.gov/my-va/welcome-va-setup/review-information/contact-information) to move to [/review-information/confirmation](https://staging.va.gov/my-va/welcome-va-setup/review-information/confirmation), nothing is announced by the screen reader. With no visual cues that content has changed, the only thing a screen reader user knows is that they've pressed a button and (as far as they can tell) nothing has happened. It's left to the user to guess what might have occurred. Did they successfully move to the next step? Is something broken? Should they try pressing the button again? Silence can be disorienting, especially for someone with a recently-acquired disability who's relatively early in their journey as a screen reader user.

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/2bd01a58-1cbb-4c2e-ad9d-1b4b54f24874

</details>

## Recommended action

This is one of those things you get for free with traditional web pages, because screen readers announce when the browser reloads on page transitions. For an SPA, you have to be a little more proactive to ensure users don't get this kind of silence on page transitions.

Typically what I would recommend here is that the page transition also moves focus to the H1 of the new page, so that H1 is immediately announced to screen reader users. In this case, the H1 is the same on the confirmation page as it was on the previous page, so sending focus there doesn't result in any new information and doesn't help the user recognize that progress has been made.

My first preference would be for the H1 to change on the confirmation page to better reflect the content of the page, and for focus ot move to that H1. If that's not possible, then the next best thing would be to send focus to the H2 in the success alert, so screen reader users get some feedback that they've successfully progressed to the next step of the flow.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by briandeconinck** [comment]
Updated with video showing the behavior. Sorry for the delay!
